common/remote/rmtfile.cpp

```diff
@@ -33,10 +33,23 @@
 
 //#define TEST_DAFILESRV_FOR_UNIX_PATHS     // probably not needed
 
+static class CSecuritySettings
+{
+    bool useSSL;
```

Comment:
CSecuritySettings::useSSL is never used, except to fill, so you may as well pass NULL to querySecuritySettings below.